packages/SketchMorph2-Core.package/M2Select.class/class/label.st

```diff
@@ -0,0 +1,3 @@
+accessing
+label
+	^ 'Select'
```

Comment:
Maybe something like `Select Area` would better represent the functionality of the button,